README.md

```diff
@@ -116,6 +116,33 @@ fun setup() {
 }
 ```
 
+JaCoCo Compatibility
+--------------------
+
+The way both JaCoCo and Paparazzi instrument may cause conflict if JDK 17+ is used. Mitigate this issue by adding the following into your JaCoCo configuration [#955](https://github.com/cashapp/paparazzi/issues/955):
```

Comment:
Maintainers: Rather than asking everyone who applies to plugins together to add this, would it be possible to add this code to PaparazziPlugin within a `plugins.withId("jacoco") {}`?

Reply:
Hmmm, this is a good idea but I wonder if it would be 100% safe to do this. Usually doing such things automatically may result into unwanted behavior.

Reply:
If we use `addAll` instead of `assign`, and maybe also wrap it in an `if (testTask.javaVersion.isCompatibleWith(JavaVersion.VERSION_17))` the risks could be mitigated and targeting the specific use case that breaks.

Reply:
So, will `Paparazzi` support this out of the box or we need to add it ourselves?